LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -669,6 +669,10 @@ namespace litecore {
 
 
     bool SQLiteDataFile::tableExists(const string &name) const {
+        if (((string_view)name).substr(0, 4) == "all_") {
+            // "all_xxx" is a TEMP VIEW that is not visible to getSchma.
+            return true;
```

Comment:
This will return false positives. Instead replace "all_" with "kv_" and then fall through to calling `getSchema`.

Reply:
Yes, agreed. Or, perhaps all_ must be used in the context of deleted. What do think falling through with  kv_del_ ?

Reply:
You can query for all deleted/non-deleted docs, even if there aren't any. So the deleted key-store shouldn't be required right?

Reply:
According to current code, we use all_ only if DeletionStatus == kLiveAndDeletedDocs, and all_ is a view that combines kv_ and kv_del_, right? In actual code consideration, we should already have checked kv_ because kv_del_ or all_ are the result of replacing aliasInfo created in the first pass, where tabelExists() already called.

Reply:
OK.